GOVERNANCE.md

```diff
@@ -45,18 +30,15 @@ be accepted unless:
   the change. Previously-objecting Collaborators do not necessarily have to
   sign-off on the change, but they should not be opposed to it.
 * The change is escalated to the CTC and the CTC votes to approve the change.
-  This should be used only after other options (especially discussion among
-  the disagreeing Collaborators) have been exhausted.
+  This should be used only after discussion among the disagreeing Collaborators
+  has been exhausted.
```

Comment:
I agree it's at least mildly awkward, but I don't think it's unclear. I'm certainly open to suggestions for better wording. Other options that spring to mind:

> This should only be used if discussion among disagreeing Collaborators fails reaches an impasse.

> This should only be as a last resort.

> This should only be used if there seems to be no other path to resolution.